NetKAN/NonAndrogynousDockingPorts.netkan

```diff
@@ -0,0 +1,18 @@
+{
+    "license": "CC-BY-NC-SA-4.0",
+    "identifier": "NonAndrogynousDockingPorts",
+    "$kref": "#/ckan/spacedock/635",
+    "spec_version": "v1.4",
+    "suggests": [
+        { "name": "ConnectedLivingSpace" }
+    ],
+    "depends": [
+        { "name": "KipEng-Agency" }
+    ],
+    "install": [
+        {
+            "find": "parts",
+            "install_to": "GameData/KipEng"
```

Comment:
If we `find KipEng` and `filter agencies` we can future-proof the install in case files or folders are added.